src/components/tutorial.ts

```diff
@@ -85,6 +89,29 @@ export class Tutorial {
 
         }
     }
+    clickOnMonsterTutorial(deltaTime) {
+        this.totalTime += Math.floor(deltaTime);
+        const transitionDuration = 1000;
+    
+        const scaleFactor = this.sinusoidalInterpolation(this.totalTime, 1, 1.5, transitionDuration);
+
+        const scaledWidth = this.tutorialImg.width * scaleFactor;
+        const scaledHeight = this.tutorialImg.height * scaleFactor;
+        const offsetX = this.endx;
+        const offsetY = this.height / 1.9 + (this.tutorialImg.height / 2);
+        this.context.drawImage(this.tutorialImg, offsetX, offsetY, scaledWidth, scaledHeight);
+    }
+    
+    sinusoidalInterpolation(time, minScale, maxScale, duration) {
+        const amplitude = (maxScale - minScale) / 2;
+        const frequency = Math.PI / duration;
+        return minScale + amplitude * Math.sin(frequency * time);
+    }
+    
```

Comment:
Can we remove unnecessary line spaces?

Reply:
Hey @Rajesh1041 i have added that line spaces for the readability, where first 2 is for declaration, next one is for function call and the last one is for the logic, purposefully i added it, should i remove that?
and one more edge case was missing here i am working on it, i will fix that and and then let you know..

Reply:
@rakshith-arch then spaces should be fine! Let me know when you are done with changes.